### ISSUE TITLE
Upgrade Expo

### ISSUE THREAD

**Issue by dansvend** [equinor/mad-project-hearing-test-client#144]
- [x] Upgrade to Expo SDK 46
- [x] Make Azure DevOps pipeline compatible with Expo SDK 46

**Comment by dansvend** [comment]
@Sayyare 

This is ready for testing 👩‍🔬

```
-----------------------------
| Platform     |        iOS |
| Version      |     0.2.13 |
| Build number | 20221124.1 |
| Environment  |       Test |
-----------------------------
```

**Comment by Sayyare** [comment]
Tested on iOS in build 0.2.13 (20221124.1), most of the functions working as expected.
It's able to take the test in on- and offline modus, but unable to see the test results, nothing shows in "Din hørsel" field when I clicked in. 
![image](https://user-images.githubusercontent.com/115469141/204277185-901f249c-1021-4fa3-9087-5db2e0ebbf7a.png)
![image](https://user-images.githubusercontent.com/115469141/204277310-be048af6-5712-4fd8-8f13-87cbb2a2837c.png)
@dansvend  Please have a look.

**Comment by dansvend** [comment]
Thank you so much for thorough testing @Sayyare 🙏

This doesn't work in production environment either, but will be implemented in #109 🙌 

I will merge this to the `main`-branch and release it to production 🚀
